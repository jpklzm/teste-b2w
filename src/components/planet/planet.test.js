import Planet from './planet';

const planet = {
    "climate": "Arid",
    "created": "2014-12-09T13:50:49.641000Z",
    "diameter": "10465",
    "edited": "2014-12-15T13:48:16.167217Z",
    "films": [
        "https://swapi.co/api/films/1/",
    ],
    "gravity": "1",
    "name": "Tatooine",
    "orbital_period": "304",
    "population": "120000",
    "residents": [
        "https://swapi.co/api/people/1/",
    ],
    "rotation_period": "23",
    "surface_water": "1",
    "terrain": "Dessert",
    "url": "https://swapi.co/api/planets/1/"
}

it("renders correctly", () => {
    const wrapper = shallow(
        <Planet planet={planet} />
    );
  
    expect(wrapper).toMatchSnapshot();
  });

it("renders correctly again", () => {
    const wrapper = render(
        <Planet planet={planet} />
    );
  
    expect(wrapper).toMatchSnapshot();
  });