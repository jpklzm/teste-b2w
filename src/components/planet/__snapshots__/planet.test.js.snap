// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="card"
  >
    <h1
      className="undefined cardTitle"
    >
      Tatooine
    </h1>
    <div
      className="cardBody"
    >
      <p>
        Population: 
        120000
      </p>
      <p>
        Climate: 
        Arid
      </p>
      <p>
        Terrain: 
        Dessert
      </p>
    </div>
    <p
      className="cardFilms"
    >
      Featured in 
      1
       films
    </p>
  </div>
</div>
`;

exports[`renders correctly again 1`] = `
<div
  class="container"
>
  <div
    class="card"
  >
    <h1
      class="undefined cardTitle"
    >
      Tatooine
    </h1>
    <div
      class="cardBody"
    >
      <p>
        Population: 120000
      </p>
      <p>
        Climate: Arid
      </p>
      <p>
        Terrain: Dessert
      </p>
    </div>
    <p
      class="cardFilms"
    >
      Featured in 1 films
    </p>
  </div>
</div>
`;
